Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -48,6 +48,41 @@ def module_test_command(client, limit, fetch_full_feed):
         return_error("Could not connect to server")
 
 
+def filter_indicators(indicators: list, last_run: dict) -> list:
```

Comment:
```suggestion
def filter_previously_fetched_indicators(indicators: list, last_run: dict) -> list:
```

Reply:
we're actually filtering the newly fetched indicators and returning new indicators list, so the word previously would be kind of misleading